### DISCUSSION TITLE
Why subtract the days in October?

### DISCUSSION THREAD

**Discussion by ChananM** [wiserdor/MilCalc#2] [General]
Why subtract the days in October?
calculator.ts@200 (it should have referenced to the line...)
![image](https://github.com/wiserdor/MilCalc/assets/19833937/33e85a10-acca-45a8-b0c3-4edaafad95c1)

**Comment by wiserdor** [comment]
Hey Chanan, After a lot of discussion with an officer which knows the calculations, we changed how the calculations are done and more. We will gradually add more parameters to the calculations.

Thank you for noticing we appreciate it alot🙏

**Reply by ChananM** [reply]
@wiserdor I actually went into the repo wanting to add a case I personally have - calculations for different service times at different units. One combatant and one not, according to the answer given here:
![image](https://github.com/wiserdor/MilCalc/assets/19833937/8de3d28a-13cb-4972-bccb-f4d82fc2e4a7)
From the file shared in my unit: [FAQ.pdf](https://github.com/wiserdor/MilCalc/files/13916068/FAQ.pdf)
But I found myself unable to understand how it applies to the 40 days threshold. If you know someone that knows how the calculations work, can you please ask him? I'll gladly add this functionality given I know how to calculate this case

**Reply by wiserdor** [reply]
sure i will ask and answer here if he will know
